server/index.js

```diff
@@ -8,6 +8,14 @@ app.use(express.static(path.join(__dirname, '../public')));
 // port 4000 for the proxy server
 const port = 4000;
 
+app.get('/:productId/:styleId', (req, res) => {
+  res.sendFile(path.join(__dirname, '../public/index.html'));
```

Comment:
How does your bundle.js gets transported with only index.html sent back to client?